spec/lib/bank_accounts/saving_spec.rb

```diff
@@ -0,0 +1,50 @@
+describe BankAccount::Saving do
+  context "creates Saving < Account"
+  
+    let (:new_account) {BankAccount::Saving.new(5,1000)}
+    let (:negative_account) {BankAccount::Saving.new(5,8)}
+    let (:penalty_account){BankAccount::Saving.new(5,23)}
+
+    it "tests method on initialize account id" do
+      expect(new_account.id).to eq(5)
+    end
+    
+    it "tests method on initial_balance" do
+      expect(new_account.initial_balance).to eq(1000)
+    end
+
+    it "raises an error if trying to start acct w/ less $10" do
+      expect{negative_account}.to raise_error(ArgumentError) 
+    end
+
+    it "withdraws money + $2 from account" do
+      expect(penalty_account.withdraw(5)).to eq(16)
+    end
+
+    it "gives a warning message if the account is 10 - 20 dollars." do
+      expect(penalty_account.withdraw(10)).to eq(11)
+    end
```

Comment:
This test deals with the value of the account but not with a warning message. Either the test needs to change, or the wording needs to be update.